source/developers-guide/event-guide/index.md

```diff
@@ -479,126 +619,55 @@ use this event to decorate / replace that SERVICE with your own one.
-{
-    public function install()
-    {
-        $this->subscribeEvent(
-            'Enlight_Controller_Dispatcher_ControllerPath_Frontend_MyPlugin',
-            'onRegisterMyPluginController'
-        );
-
-
-        return true;
-    }
-
-    public function onRegisterMyPluginController(Enlight_Event_EventArgs $args)
-    {
-        Shopware()->Template()->addTemplateDir(__DIR__ . '/../Views');
-        return __DIR__ . '/../Controllers/Frontend/MyController.php';
-    }
-}
-```
-
-This will register a controller for the route `frontend/my_plugin`. Notice that in the event subscriber a template
-directory is registered as well. This is useful, as it makes sure, that all your templates are available in the controller -
-so the automatic template loading will work. Also notice, that the event subscriber needs to return the controller path.
-
-For a more convenient way to register a controller, see [the controller documentation](/developers-guide/controller/).
-
-### Registering a service
-```
-class Shopware_Plugins_Frontend_SwagMyPlugin_Bootstrap extends Shopware_Components_Plugin_Bootstrap
-{
-    public function install()
-    {
-        $this->subscribeEvent(
-            'Enlight_Bootstrap_InitResource_swag_myplugin.my_service',
-            'onLoadMyService'
-        );
-
-
-        return true;
-    }
-
-    public function onLoadMyService(Enlight_Event_EventArgs $args)
-    {
-        return new MyService();
-    }
-}
-```
-
-This example shows, how to register a service in Shopware. As soon as the service `swag_myplugin.my_service` is requested
-from the DI container, Shopware will emit the event `Enlight_Bootstrap_InitResource_swag_myplugin.my_service` so you can
-register to it and return an *instance* of your service. See [the service documentation](/developers-guide/services/) for
-more details.
-
-
-### Decorating / replacing a service
-```
-class Shopware_Plugins_Frontend_SwagMyPlugin_Bootstrap extends Shopware_Components_Plugin_Bootstrap
-{
-    public function install()
-    {
-        $this->subscribeEvent(
-            'Enlight_Bootstrap_AfterInitResource_some_service',
-            'onDecorateSomeService'
-        );
-
-
-        return true;
-    }
-
-    public function onDecorateSomeService(Enlight_Event_EventArgs $args)
-    {
-        return new MyService(
-            Shopware()->Container()->get('some_service')
-        );
-    }
-}
-```
-In this example the event `Enlight_Bootstrap_AfterInitResource_some_service` is used, to decorate the service `some_service`
-with `MyService`. See the [core service extension documentation](/developers-guide/shopware-5-core-service-extensions/#your-extension-plugin)
-for a more detailed explanation.
 
+Controllers are loaded automatically if you use the directories `PluginBaseDirectory/Controller/(Frontend | Backend | Api | Widget)/your_controller.php`,
+registering controllers are so not necessary.
 
+Notice that in the event subscriber a template directory is registered as well. 
```

Comment:
I think these two sentences are absolete as you do not show the controller events anymore